arch/arm/soc/rda_5981/Kconfig

```diff
@@ -0,0 +1,23 @@
+#
+# Copyright (c) 2017 RDA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config SOC_FAMILY_RDA5981
+	bool
+	# omit prompt to signify a "hidden" option
+	default n
+
+if SOC_FAMILY_RDA5981
+config SOC_FAMILY
+	string
+	default rda_5981
+
+config SOC_RDA_DEBUG
+	bool
+	prompt "Use for RDA SOC DBG"
+	default n
```

Comment:
It is not hardware debug.If select Y here, you can  print more debugging messages.